### ISSUE TITLE
Text layer shift

### ISSUE THREAD

**Issue by pierrebonny** [VadimDez/ng2-pdf-viewer#596]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

On text selection there is a white shift appearing because text layer is shifted. How can I fix it ?
Thanks for your work
![shift](https://user-images.githubusercontent.com/23218394/78375900-ca3d9380-75cd-11ea-8e69-7057ff4c48fe.png)

**Comment by souravprakashjena** [comment]
@pierrebonny  did you find any solution?

**Comment by pierrebonny** [comment]
More or less, I finally had to use custom css inspired by the one used in mozilla's pdf js library for this problem

**Comment by souravprakashjena** [comment]
@pierrebonny  thanks

**Comment by VadimDez** [comment]
This should be fixed with version `6.3.1`
